server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftClientHandler.java

```diff
@@ -1433,8 +1432,11 @@ public void loadTablet(TInfo tinfo, TCredentials credentials, String lock,
             all.remove(extent);
 
             if (!all.isEmpty()) {
-              log.error("Tablet {} overlaps previously assigned {} {} {}", extent,
-                  unopenedOverlapping, openingOverlapping, onlineOverlapping + " " + all);
+              log.error("Tablet {} overlaps a previously assigned tablet. It is possibly due to a recent split " +
+                        "(though it tries to ignore them as a reason to throw this error.) Possible overlapping tablets:\n" +
+                        "Unopened tablet:  {}\n" +
+                        "Opening tablet:   {}\n" +
+                        "Online tablet:    {}", extent, unopenedOverlapping, openingOverlapping, onlineOverlapping);
```

Comment:
```suggestion
              log.error("Tablet {} overlaps a previously assigned tablet. It is possibly due to a recent split. " +
                        "Possible overlapping tablets:\n" +
                        "Unopened tablet:  {}\n" +
                        "Opening tablet:   {}\n" +
                        "Online tablet:    {}", extent, unopenedOverlapping, openingOverlapping, onlineOverlapping);
```

I am not quite sure what the parenthesis section was adding and I don't think it is necessary to this log message.